std_msgs/Header header
float64 ship_longitude            # ship_longitude reading
float64 ship_longitude_variance   # ship_longitude_variance reading

